internal/server/testdata/openapi3.json

```diff
@@ -865,7 +865,6 @@
             }
           },
           {
-            "example": "4yJ3n3D8E2",
```

Comment:
We had these examples in two places. If you expand the fold below this section, you should see all of these have a `schema.example` with the same value.

Reading the docs here https://swagger.io/specification/#fixed-fields-10 (second table) it says:

> Furthermore, if referencing a schema that contains an example, the example value SHALL override the example provided by the schema.

So we only need to set this example if we are overriding the `schema.example`.